test/js-native-api/test_typedarray/test_typedarray.c

```diff
@@ -97,6 +97,14 @@ static napi_value External(napi_env env, napi_callback_info info) {
   return output_array;
 }
 
+
+static napi_value EmptyArrayBuffer(napi_env env, napi_callback_info info) {
+  void* null_data = NULL;
```

Comment:
`nullptr` might be preferred.

Reply:
Yeah but this is strictly `C` so there is no `nullptr`.